packages/manager/modules/pci/src/projects/project/users/roles-matrix/roles-matrix.html

```diff
@@ -0,0 +1,108 @@
+<div class="d-none d-md-block">
+    <div class="oui-datagrid-topbar__actions">
+        <div class="oui-criteria-adder oui-datagrid-parameters mb-2">
+            <oui-dropdown data-align="end" data-arrow data-persistent>
+                <button class="oui-datagrid-parameters__button oui-button oui-button_secondary oui-button_small-width" oui-dropdown-trigger>
+                    <span class="oui-icon oui-icon-gear" aria-hidden="true"></span>
+                    <span class="sr-only" data-translate="pci_projects_project_users_roles_matrix_customize"></span>
+                </button>
+                <oui-dropdown-content>
+                    <div class="oui-datagrid-parameters__body pt-1">
+                        <div
+                            class="oui-datagrid-parameters__column"
+                            data-ng-repeat="role in $ctrl.roles.roles|orderBy:name track by role.id">
+                            <div class="oui-datagrid-parameters__column-column">
+                                <oui-checkbox data-model="$ctrl.columns[role.id]">
+                                    <span data-ng-bind=":: role.name"></span>
+                                </oui-checkbox>
+                            </div>
+                        </div>
+                    </div>
+                </oui-dropdown-content>
+            </oui-dropdown>
+        </div>
+    </div>
+    <div class="users-roles-matrix oui-datagrid-responsive-container oui-datagrid-responsive-container_sticky-checkbox">
+        <div class="oui-datagrid-responsive-container__sticky-container oui-datagrid-responsive-container__sticky-container_scroll-begin oui-datagrid-responsive-container__sticky-container_scroll-end">
+            <div class="oui-datagrid-responsive-container__overflow-container">
+                <table class="oui-datagrid">
+                    <thead class="oui-datagrid__headers">
+                        <tr>
+                            <th class="oui-datagrid__header oui-datagrid__cell-sticky users-roles-matrix__service"></th>
```

Comment:
```suggestion
                            <th  scope="col" class="oui-datagrid__header oui-datagrid__cell-sticky users-roles-matrix__service"></th>
```

Same for all the others